tests/stack/control-flow.wast

```diff
@@ -0,0 +1,123 @@
+// Blocks
+
+block [ i32 i32 i32 ]
+    i32.const 1
+    i32.const 2
+    i32.const 3
+end
+#assertStack < i32 > 3 : < i32 > 2 : < i32 > 1 : .Stack "block 1"
+
+block [ i32 i32 ]
+    i32.const 1
+    i32.const 2
+    i32.const 3
+    drop
+end
+#assertStack < i32 > 2 : < i32 > 1 : .Stack "block 2"
+
+block [ i32 i32 ]
+    i32.const 1
+    i32.const 2
+    i32.const 3
+end
+#assertStack < i32 > 3 : < i32 > 2 : .Stack "block 3 (invalid)"
```

Comment:
Should also add a stack underflow case:
```
block [ i32 i32 ]
   i32.const 1
end
```

Reply:
I've added a TODO, and an issue https://github.com/kframework/wasm-semantics/issues/11